onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -18,9 +18,7 @@
 if platform.system() == 'Windows' and sys.version_info.major >= 3 and sys.version_info.minor >= 8:
     os.add_dll_directory(os.getcwd())
 
-available_providers = [
-    provider for provider in onnxrt.get_available_providers()
-    if provider not in {'TvmExecutionProvider'}]
+available_providers = [provider for provider in onnxrt.get_available_providers()]
```

Comment:
Why not having two lists? One with TvmExecutionProvider, one without for the test with symbolic shapes?

Reply:
Hello @xadupre!
In this case, for each of the tests it will be necessary to indicate which list of providers to use. It seems that this is not very convenient, because you have to store the number of lists equal (or almost equal) to the number of tests.
Also, using one list with excluded providers will not work, because some tests must exclude some providers, other tests - other providers.
In addition, the `if` approach is already used for [TVM](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L36), [CUDA](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L72), [TensorRT](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L82), [CUDA](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L141), [CUDA](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L890), [CUDA](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L905), [CUDA](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L965), [CUDA](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L1024), [ROCM](https://github.com/microsoft/onnxruntime/blob/8d06e5a9df3024dbaadf4e221812d2fd46faabcd/onnxruntime/test/python/onnxruntime_test_python.py#L1113).

Reply:
I suggested having two lists. The goal is to keep running the test without Tvm provider instead of skipping it.

Reply:
done